deploy/job-manager/job-manager-deployment.yaml.ctmpl

```diff
@@ -22,22 +22,28 @@ spec:
         ports:
         - containerPort: 8190
         volumeMounts:
+        {{ if not (env "USE_CAAS" | parseBool) }}
         - name: jm-api-config
           mountPath: /etc/job-manager/api
           readOnly: true
+        {{ end }}
         - name: jm-api-capabilities-config
           mountPath: /etc/job-manager/capabilities/capabilities-config.json
           subPath: capabilities-config
           readOnly: true
         env:
         - name: PATH_PREFIX
           value: {{ env "API_PATH_PREFIX" }}
+        {{ if not (env "USE_CAAS" | parseBool) }}
         - name: CROMWELL_CREDENTIALS
           value: /etc/job-manager/api/config.json
+        {{ end }}
         - name: CROMWELL_URL
           value: {{ env "CROMWELL_URL" }}
         - name: CAPABILITIES_CONFIG
           value: /etc/job-manager/capabilities/capabilities-config.json
+        - name: USE_CAAS
+          value: "{{ env "USE_CAAS" }}"
       - name: job-manager-ui
         image: {{ env "UI_DOCKER_IMAGE" }}
         ports:
```

Comment:
Add `imagePullPolicy: Always` below the image to prevent GKE from evilly(sometimes) remembering the old images.